ngsild-payloads/trafficFlowObserved.jsonld

```diff
@@ -1,95 +1,146 @@
-        "predictedBy": "urn:ngsi-ld:DataService:XXXX",
-        "vehicleType": {
-            "type": "Property",
-            "value": "All"
-        }
-    },{
-        "type": "Property",
-        "observedAt": "2023-11-29T08:00:00Z",
-        "value": 86.8995985327,
-        "unitCode": "E50",
-        "datasetId": "urn:ngsi-ld:Dataset:LightVehicle",
-        "predictedBy": "urn:ngsi-ld:DataService:YYYY",
-        "vehicleType": {
-            "type": "Property",
-            "value": "Light Vehicle"
-        }
-    }],
-    "flow": [{
+    {
+      "type": "Property",
+      "observedAt": "2023-11-29T08:00:00Z",
+      "value": 86.8995985327,
+      "unitCode": "E50",
+      "datasetId": "urn:ngsi-ld:Dataset:LightVehicle",
+      "vehicleType": {
         "type": "Property",
-        "observedAt": "2023-11-29T08:00:00Z",
-        "value": 97.0749588766,
-        "unitCode": "E50",
-        "datasetId": "urn:ngsi-ld:Dataset:All",
-        "vehicleType": {
-            "type": "Property",
-            "value": "All"
-        }
-    },{
-        "type": "Property",
-        "observedAt": "2023-11-29T08:00:00Z",
-        "value": 86.8995985327,
-        "unitCode": "E50",
-        "datasetId": "urn:ngsi-ld:Dataset:LightVehicle",
-        "vehicleType": {
-            "type": "Property",
-            "value": "Light Vehicle"
-        }
-    },{
+        "value": "Light Vehicle"
+      }
+    },
+    {
+      "type": "Property",
+      "observedAt": "2023-11-29T08:00:00Z",
+      "value": 8.4055710091,
+      "unitCode": "E50",
+      "datasetId": "urn:ngsi-ld:Dataset:CommercialVehicle",
+      "vehicleType": {
         "type": "Property",
-        "observedAt": "2023-11-29T08:00:00Z",
-        "value": 8.4055710091,
-        "unitCode": "E50",
-        "datasetId": "urn:ngsi-ld:Dataset:CommercialVehicle",
-        "vehicleType": {
-            "type": "Property",
-            "value": "Commercial Vehicle"
-        }
-    },{
+        "value": "Commercial Vehicle"
+      }
+    },
+    {
+      "type": "Property",
+      "observedAt": "2023-11-29T08:00:00Z",
+      "value": 1.7697893348,
+      "unitCode": "E50",
+      "datasetId": "urn:ngsi-ld:Dataset:HeavyWeight",
+      "vehicleType": {
         "type": "Property",
-        "observedAt": "2023-11-29T08:00:00Z",
-        "value": 1.7697893348,
-        "unitCode": "E50",
-        "datasetId": "urn:ngsi-ld:Dataset:HeavyWeight",
-        "vehicleType": {
-            "type": "Property",
-            "value": "Heavy Weight"
-        }
-    }],
-    "co2Equivalent": {
+        "value": "Heavy Weight"
+      }
+    }
+  ],
+  "co2Equivalent": {
+    "type": "Property",
+    "observedAt": "2023-11-29T08:00:00Z",
+    "value": 0.0113340287,
+    "unitCode": "TNE"
+  },
+  "simulatedFlow": [
+    {
+      "type": "Property",
+      "value": 61.0107606665808,
+      "observedAt": "2024-05-14T23:00:00Z",
+      "unitCode": "E50",
+      "simulatedBy": "urn:ngsi-ld:DataService:ZZZZ",
```

Comment:
@junjou from TP's point of view, wouldn't it be easier to define a generic relationship name here?

Reply:
You mean to do something like that instead ?
```json
"simulatedBy": {
  "type": "Relationship",
  "object": "urn:ngsi-ld:DataService:ZZZZ"
}
```
Also probably the same is worth for `predictedBy`

Reply:
no, to use the same relationship name whether it is a simulation or a prediction (something like "processedBy" for instance). to allow for a generic handling on the front side.

Reply:
Shall I change for both predictedBy and simulatedBy

`"processedBy": "urn:ngsi-ld:DataService:ZZZZ"`

Reply:
OK, then yes 

> no, to use the same relationship name whether it is a simulation or a prediction (something like "processedBy" for instance). to allow for a generic handling on the front side.

ok, then yes it would be easier to have a single relationship name